src/drawers/errorBarDrawer.ts

```diff
@@ -0,0 +1,68 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module _Drawer {
+  export class ErrorBar extends Element {
```

Comment:
As an aside, if you can think of a good deviating architecture for Drawers, feel free to use it.  Drawers are right now in a bit of a funky state so a good deviating architecture can be used as a precedent possibly